README.md

```diff
@@ -78,3 +78,12 @@ default:
 contacts-ui:
   port: 8888
 ```
+
+Multiple Configurations
+-----------------------
+
+You may want to have multiple configuration files within the same project, in order to allow different proxying rules depending on the envirnoment you are working on (e.g. local vs QA). You can specify a custom config file by using the `--config` flag as follows:
+
+```bash
+sudo vee --config .vee.qa
```

Comment:
It's probably not necessary to imply that sudo is needed for vee. That's only if your port is a reserved port.